guides/common/modules/proc_configuring-tls-for-secure-ldap.adoc

```diff
@@ -1,26 +1,21 @@
-[id='configuring-tls-for-secure-ldap_{context}']
+[id="Configuring_TLS_for_Secure_LDAP_{context}"]
 = Configuring TLS for Secure LDAP
 
 Use the {Project} CLI to configure TLS for secure LDAP (LDAPS).
 
 .Procedure
-
 . Obtain the Certificate from the LDAP Server.
-
 .. If you use Active Directory Certificate Services, export the Enterprise PKI CA Certificate using the Base-64 encoded X.509 format.
 ifndef::orcharhino[]
 See https://access.redhat.com/solutions/1498773[How to configure Active Directory authentication with `TLS` on {Project}] for information on creating and exporting a CA certificate from an Active Directory server.
 endif::[]
-
-.. Download the LDAP server certificate to a temporary location on the Red{nbsp}Hat Enterprise{nbsp}Linux system where {ProjectServer} is installed and remove it when finished.
+.. Download the LDAP server certificate to a temporary location onto {ProjectServer} and remove it when finished.
 +
 For example, `/tmp/example.crt`.
 The filename extensions `.cer` and `.crt` are only conventions and can refer to DER binary or PEM ASCII format certificates.
-
 . Trust the Certificate from the LDAP Server.
 +
 {ProjectServer} requires the CA certificates for LDAP authentication to be individual files in `/etc/pki/tls/certs/` directory.
-
```

Comment:
This deletion causes problems in the rendering..

Reply:
Fixed.